### PR TITLE
Fix qt4 build

### PR DESCRIPTION
**Pull request by ogoffart**

### CODE REVIEW

**Comment by ogoffart** [comment]
@danimo  for the CMakeLists change,  @dragotin for the ActivityWidget changes

File: src/gui/activitywidget.cpp [hunk 1]
```diff
@@ -440,32 +442,35 @@ void ActivityWidget::scheduleWidgetToRemove(NotificationWidget *widget, int mill
...
-
-    QPair<QDateTime, NotificationWidget*> removeInfo = qMakePair(removeTime, widget);
-    if( !_widgetsToRemove.contains(removeInfo) ) {
-        _widgetsToRemove.insert( removeInfo );
-        if( !_removeTimer.isActive() ) {
-            _removeTimer.start();
-        }
+    // in five seconds from now, remove the widget.
+    QDateTime removeTime = QDateTime::currentDateTimeUtc().addMSecs(milliseconds);
+    QDateTime &it = _widgetsToRemove[widget];
```
Comment (dragotin) [inline src/gui/activitywidget.cpp]: Minor: Usually we use the variable name it for iterators, here it's a DateTime

**Comment by dragotin** [comment]
Thanks @ogoffart for cleaning up :+1: 
Why do we do the CMake changes?

**Comment by ogoffart** [comment]
without the cmake changes, the dolphin plugin does not compile because the Qt4 include paths are injected. That's why i moved the Qt detection to src/

**Pull request merged**